src/topology/continuous_function/basic.lean

```diff
@@ -183,6 +183,9 @@ def prod_map (f : C(α₁, α₂)) (g : C(β₁, β₂)) :
 @[simp] lemma prod_eval (f : C(α, β₁)) (g : C(α, β₂)) (a : α) :
   (prod_mk f g) a = (f a, g a) := rfl
 
+@[simp] lemma prod_map_eval (f : C(α₁, α₂)) (g : C(β₁, β₂)) (x : α₁) (y : β₁) :
```

Comment:
Instead of adding this lemma, I'd be inclined to just put `@[simps]` on the `prod_map` definition. Can you check (using `@[simps?]`) that that does something reasonable?

Reply:
Here's the output:
`[simps] > adding projection continuous_map.prod_map_to_fun:
        > ∀ {α₁ : Type u_5} {α₂ : Type u_6} {β₁ : Type u_7} {β₂ : Type u_8}
[_inst_5 : topological_space α₁] [_inst_6 : topological_space α₂] [_inst_7 : topological_space β₁]
[_inst_8 : topological_space β₂] (f : C(α₁, α₂)) (g : C(β₁, β₂)) (x : α₁ × β₁),
  ⇑(f.prod_map g) x = prod.map ⇑f ⇑g x
`

It does work for my purposes